### PR TITLE
fix 212

### PR DESCRIPTION
**Pull request by ocefpaf**
@BibMartin this is far from ideal because I am encoding only when there is no `self.json_data`.  But I do not want to hold `v0.1.6` and I want all our efforts to be on `v0.2.0`.

### CODE REVIEW

File: folium/folium.py [hunk 1]
```diff
@@ -802,9 +802,9 @@ def geo_json(self, geo_path=None, geo_str=None, data_out='data.json',
             keyword argument will enable conversion to GeoJSON.
         reset: boolean, default False
             Remove all current geoJSON layers, start with new layer
-        freescale: if True use free format for the scale, where min and max values
-            are taken from the data. It also allow to plot allow to plot values < 0 
-            and float legend labels.
+        freescale: if True use free format for the scale, where min and max
+        values are taken from the data. It also allow to plot allow to plot
+        values < 0 and float legend labels.
```
Comment (BibMartin) [inline folium/folium.py]: I guess there's a typo here, isn't it ?
 you may keep the indent, and avoid the duplication of `allow to plot`

**Comment by BibMartin** [comment]
Except the line 805-807 comment changes, I'm okay for the rest.
Merge soon

File: folium/folium.py (hunk 1 above)
Comment (ocefpaf) [inline folium/folium.py]: Urgh!  My pep8 stupid mindset attacks again.

**Comment by BibMartin** [comment]
Perfect. Squash ?

**Comment by BibMartin** [comment]
Perfect. Squash ?

**Comment by ocefpaf** [comment]
Done.

**Comment by BibMartin** [comment]
Merged. :gun:

**Comment by ocefpaf** [comment]
`v0.1.6` is almost out of the windows :relieved:

**Comment by ocefpaf** [comment]
I am rethinking this one and I plan to revert it.

This is a bad solution and we should just tell to people to start migrating to `v0.2.0`.

The problems are:
- It is inconsistent.  I encode the HTML only when it does not have `json_data`. I should've used `self.map_type = ='geojson'` instead.
- It breaks PNG, Fancy HTML (mpld3), and Vega popups.

I guess the downsides are not worth fixing #212.

**Pull request merged**